generators/profilegen/src/resources/dir_metadata.json

```diff
@@ -0,0 +1,292 @@
+{
+  "Microsoft.AnalysisServices": {
+    "path": "../../../management/azure_mgmt_analysis_services/",
```

Comment:
QQ: seems like we implicitly rely on the `path` value to be ended with `/`? If so, please make note on this where we are using this value. If not replying then that's great :)

Reply:
Removed it